### PR TITLE
[WIP] Add Courses List

### PR DESCRIPTION
**Pull request by daviddias**
- [ ] Confirm course titles
- [x] Add remaining information
- [x] Create placeholder files for each course

### CODE REVIEW

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 1]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
 
 The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level-up each participant's understanding so that we all share a great baseline to build upon. The Elective Courses will cover even more bleeding-edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge and deepen their understanding.
 
-The courses list is: `TBA`
+The courses list is:
+
+|                | Title  | Trainers  | Short Description  | Link  |
+|--------------- |------- |---------- |------------------- |------ |
+| **Core**       |        |           |                    |       |
+| A              | Understanding how the InterPlanetary File System deals with Files       |           |                    | [CORE_COURSE_A](CORE_COURSE_A) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @alanshaw @Stebalien @mikeal, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 2]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
 The Courses are intensive workshops of 75 minutes each. The Core Courses will be a series of sessions to level-up each participant's understanding so that we all share a great baseline to build upon. The Elective Courses will cover even more bleeding-edge topics of the DWeb, where each participant decides which ones to attend to advance their knowledge and deepen their understanding.
 
-The courses list is: `TBA`
+The courses list is:
+
+|                | Title  | Trainers  | Short Description  | Link  |
+|--------------- |------- |---------- |------------------- |------ |
+| **Core**       |        |           |                    |       |
+| A              | Understanding how the InterPlanetary File System deals with Files       |           |                    | [CORE_COURSE_A](CORE_COURSE_A) |
+| B              | Processing Addressing wut? A packet epic adventure through the Internet       |           |                    | [CORE_COURSE_B](CORE_COURSE_B) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @raulk @jacobheun @achingbrain, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 3]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| C              | Developing Apps with IPFS. Pick implementation and way of interacting with it       |           |                    | [CORE_COURSE_C](CORE_COURSE_C) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @lidel @jimpick @hugomrdias, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 4]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| D              | The lifecycle of data in DWeb.       |           | Providing (Share), Pinning (Store) and Blocking (Delete) Data. How IPFS answers all of these questions.                   | [CORE_COURSE_D](CORE_COURSE_D) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @olizilla @kishansagathiya @meiqimichelle, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 5]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| **Electives**  |        |           |                    |       |
+| A              | Identity on the DWeb       |           |                    | [ELECTIVE_COURSE_A](ELECTIVE_COURSE_A) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @satazor @paulobmarcos @joaosantos15, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 6]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| B              | Building with IPFS Cluster       |           |                    | [ELECTIVE_COURSE_B](ELECTIVE_COURSE_B) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @hsanjuan @lanzafame @meiqimichelle, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 7]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| C              | TBA       |           |                    | [ELECTIVE_COURSE_C](ELECTIVE_COURSE_C) |
+
+| D              | Building DApps with the iCloud for the DWeb, Textile       |           |                    | [ELECTIVE_COURSE_D](ELECTIVE_COURSE_D) |
+
+| E              | Deploying IPFS Infrastructure       |           |                    | [ELECTIVE_COURSE_E](ELECTIVE_COURSE_E) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @mburns, @obo20, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 8]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| F              | TBA       |           |                    | [ELECTIVE_COURSE_F](ELECTIVE_COURSE_F) |
+
+| G              | Managing Datasets with QRI       |           | An open source tool for versioned, scriptable, exportable, collaborative datasets                   | [ELECTIVE_COURSE_G](ELECTIVE_COURSE_G) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @b5, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 9]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+
+| H              | Make the Browser You Want to See in the World (DWeb Browser Hacking)       |           |                    | [ELECTIVE_COURSE_H](ELECTIVE_COURSE_H) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @autonome, @gozala, @soapdog, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md [hunk 10]
```diff
@@ -6,7 +6,35 @@ The Core and Elective Courses are the main sessions for the InterPlanetary Space
...
+| D              | The lifecycle of data in DWeb.       |           | Providing (Share), Pinning (Store) and Blocking (Delete) Data. How IPFS answers all of these questions.                   | [CORE_COURSE_D](CORE_COURSE_D) |
+
+| **Electives**  |        |           |                    |       |
+| A              | Identity on the DWeb       |           |                    | [ELECTIVE_COURSE_A](ELECTIVE_COURSE_A) |
+
+| B              | Building with IPFS Cluster       |           |                    | [ELECTIVE_COURSE_B](ELECTIVE_COURSE_B) |
+
+| C              | TBA       |           |                    | [ELECTIVE_COURSE_C](ELECTIVE_COURSE_C) |
+
+| D              | Building DApps with the iCloud for the DWeb, Textile       |           |                    | [ELECTIVE_COURSE_D](ELECTIVE_COURSE_D) |
```
Comment (daviddias) [inline CORE_AND_ELECTIVE_COURSES/README.md]: @balupton, @carsonfarmer, @andrewxhill, can you confirm that the course Title is still accurate?

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 1 above)
Comment (alanshaw) [reply CORE_AND_ELECTIVE_COURSES/README.md]: :thumbsup:

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 3 above)
Comment (lidel) [reply CORE_AND_ELECTIVE_COURSES/README.md]: Given the current course outline I would tweak it a bit:

```suggestion
| C              | Developing Apps with IPFS API: pick implementation and a way of interacting with it       |           |                    | [CORE_COURSE_C](CORE_COURSE_C) |
```

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 4 above)
Comment (olizilla) [reply CORE_AND_ELECTIVE_COURSES/README.md]: 👍

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 8 above)
Comment (b5) [reply CORE_AND_ELECTIVE_COURSES/README.md]: Looks good to me! Thanks @daviddias 😄

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 5 above)
Comment (satazor) [reply CORE_AND_ELECTIVE_COURSES/README.md]: @paulobmarcos @joaosantos15 do you want to change the title to something more fancy or is this sufficient?

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 4 above)
Comment (daviddias) [reply CORE_AND_ELECTIVE_COURSES/README.md]: Thank you for your _thumb pointing up of confirmation_

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 10 above)
Comment (carsonfarmer) [reply CORE_AND_ELECTIVE_COURSES/README.md]: Updated directly in branch... hope that's ok?

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 5 above)
Comment (paulobmarcos) [reply CORE_AND_ELECTIVE_COURSES/README.md]: I think it's right 👍

**Comment by daviddias** [comment]
Going ahead and merging this one as is so that trainers have the placeholders to PR their training materials to :)

File: CORE_AND_ELECTIVE_COURSES/README.md (hunk 5 above)
Comment (joaosantos15) [reply CORE_AND_ELECTIVE_COURSES/README.md]: Yup, sounds good!

**Pull request merged**